src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/AesGcm.Windows.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Internal.Cryptography;
+using Internal.NativeCrypto;
+using static Interop.BCrypt;
+
+namespace System.Security.Cryptography
+{
+    partial class AesGcm
+    {
+        private static readonly SafeAlgorithmHandle s_aesGcm = AesBCryptModes.OpenAesAlgorithm(Cng.BCRYPT_CHAIN_MODE_GCM);
+        private SafeKeyHandle _keyHandle;
+        public static KeySizes TagByteSizes { get; } = AesAEAD.GetTagLengths(s_aesGcm);
```

Comment:
Why bother asking the OS for the answer we already know?  I'd just make this be defined as a literal in the AnyOS version.

Reply:
I think this is implementation-specific

Reply:
IIRC, Windows only allows a single size here, and we locked the OpenSSL/Unix one to the single Windows value.

Reply:
@bartonjs I was wondering if this should be hard coded or "always ask OS" - will change to hard-coded as it probably won't change in the future